static/scss/calculator.scss

```diff
@@ -81,3 +81,56 @@
     }
   }
 }
+
+@include breakpoint(phone) {
+  .financial-aid-calculator, .confirm-dialog {
```

Comment:
one of the nice things (I think!) about the `breakpoint` mixin is it allows you to easily use breakpoints all over the place, for instance, my preferred way to use them is to set styles on specific things, so I might have something like

```scss
.my-component-class {
  .buttons {
    .first-button {
      color: blue,
      font-size: 12px;
      
      @include breakpoint(phone) {
        margin: 12px;
      }
      @include breakpoint(desktop) {
        margin: 24px;
      }
    }
  }
}
```

instead of having a big block for mobile code which declares all of the same selectors. does that make sense? IMO that approach makes the CSS easier to change, for instance if the class `.first-button` gets refactored to `.save-button` we only need to change it in one place.